game/hud/src/components/Watermark.tsx

```diff
@@ -32,5 +32,5 @@ export interface WatermarkStyle {
 export default (props: {
   style?: Partial<WatermarkStyle>;
 }) => {
-  return <Watermark>Beta 1 - Do not stream or distribute.</Watermark>;
+  return <Watermark style={props.style ? props.style.watermark : {}}>Beta 1 - Do not stream or distribute.</Watermark>;
```

Comment:
We should add a `pointer-events: none` to this guy so it doesn't block certain UI's (and there's no point it needs to capture mouse events). I can't click on the Maximize or X button on this widget.
![cantclick](https://user-images.githubusercontent.com/14223893/43737735-492d13c6-9990-11e8-8868-da11e070bbb7.PNG)

Reply:
I have updated watermark to disable pointer events